data/json/professions.json

```diff
@@ -5324,5 +5324,49 @@
                 "panties"
             ]
         }
+    },   
+    {
+        "type": "profession",
+        "ident": "cyclist_joust",
+        "name": "Bicycle Jouster",
+        "description": "A vagrant before the cataclysm, you had settled down in an underground jousting ring. Now, that ring is full of undead.",
+        "points": 4,
+        "skills": [
+            {
+                "level": 3,
+                "name": "driving"
+            },
+            {
+                "level": 3,
+                "name": "stabbing"
+            }
```

Comment:
missing separator after "stabbing" skill entry, and after "dodge" skill entry. Causing errors in build systems.